R/Species.r

```diff
@@ -0,0 +1,97 @@
+
+#' Infer Species
+#'
+#' @param sset a \code{SigSet}
+#' @param df_as a data.frame of alignment score for each probe.
+#' @param hm.pos.por whether to use the postive probes portion to infer human.
+#' @param topN Top n positive and negative probes used to infer species.
+#' @param plot Whether to plot ROC curve.
+#' @return species (NCBI official species names)
+#' We infer species based on probes pvalues and alignment score.
+#' We calculate AUC for each species, y_true is whether pval < 0.01
+#' y_pred is normalized alignment score (AS / 50).
+#'
+#' Our function works on a single sample.
+#' @importFrom PRROC roc.curve
+#' @import sesameData
+#' @examples
+#' sset <- sesameDataGet('EPIC.1.LNCaP')$sset
+#' df_as <- sesameDataGet() #TODO
+#' inferSpecies(sset)
+#' 
+#' @examples
+#' library(tidyverse)
+#' library(sesame)
+#' library(parallel)
+#' library(readxl)
+#' indir="/mnt/isilon/zhou_lab/projects/20191212_GEO_datasets/IDATs/Mammal40/"
```

Comment:
we cannot use these local paths

Reply:
Which version were you looking at? This was the very old version.